bin/benchmark

```diff
@@ -5,10 +5,12 @@
 $LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
 $VERBOSE = nil
 
+require "bundler/setup"
 require "ruby_lsp/internal"
 require "benchmark"
 
-ITERATIONS = 50
+ITERATIONS = 500
```

Comment:
Had to bump the number of iterations to get a more reliable average. Using 50 iterations made the script always fail because there was also some variance.